service/storage_proxy.cc

```diff
@@ -3203,9 +3205,8 @@ static inet_address_vector_replica_set endpoint_filter(const gms::gossiper& g, c
     // strip out dead endpoints and localhost
     std::unordered_multimap<sstring, gms::inet_address> validated;
 
-    auto is_valid = [&g] (gms::inet_address input) {
-        return input != utils::fb_utilities::get_broadcast_address()
-            && g.is_alive(input);
+    auto is_valid = [&is_alive] (gms::inet_address input) {
+        return input != utils::fb_utilities::get_broadcast_address() && is_alive(input);
```

Comment:
nit: please refrain with making functional and cosmetic changes at the same time.
It makes reviewing harder

Reply:
nit2: the `input != utils::fb_utilities::get_broadcast_address()` part is excessive. The `gossiper::is_alive()` reports true for bcast address value